packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -69,39 +117,53 @@ class MouseTrackerAnnotation {
 /// position.
 typedef MouseDetectorAnnotationFinder = Iterable<MouseTrackerAnnotation> Function(Offset offset);
 
+typedef _UpdatedDeviceHandler = void Function(_MouseState mouseState, LinkedHashSet<MouseTrackerAnnotation> previousAnnotations);
+
 // Various states of each connected mouse device.
 //
-// It is used by [MouseTracker] to compute which callbacks should be triggered
-// by each event.
+// It is used by [MouseTracker] to record the current state, as well as the
```

Comment:
These don't terms don't need square brackets, since they're not in a dartdoc comment. (and elsewhere in the file).